utilities/sourceEvents.py

```diff
@@ -270,16 +270,8 @@ def parse(content, gmaps):
             entry['endDate'] = event_time_conversion.utctime(endDateObj, entry_location.get('latitude', 40.1153287), entry_location.get('longitude', -88.2280659))
 
         if pe['timeType'] == "ALL_DAY":
```

Comment:
I think this check for "ALL_DAY" can go as the first thing to check for when entering the for loop. Because at line 272, half of the fields are put in "entry" which is not required.
I think the "ALL_DAY" check can go [here](https://github.com/rokwire/events-manager/blob/a913fadc9f3d7cd61a725980fddb8382fec52044/utilities/sourceEvents.py#L132) , similar to check for "shareWithIllinoisMobileApp"

Reply:
LGTM. Resolving